mediator/src/MediatorPatternClasssDiagramTemplate.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+'https://plantuml.com/class-diagram
+
+interface Mediator {
+    sendMessage(message: String): void
+}
+
+class MediatorImpl implements Mediator{
+    - colleagues: List<Colleague>
+
+    + sendMessage(message: string, sender: Colleague): void
+}
+
+interface Colleague {
```

Comment:
nit: Communication or some other naming convention may be meaningful here.

Reply:
done